scripts/qa_crowbarsetup.sh

```diff
@@ -3698,6 +3698,9 @@ function oncontroller_testsetup
 {
     . .openrc
     oncontroller_prepare_functional_tests
+    if [[ $hacloud = 1 ]] ; then
+        crm_mon --failcounts -1 | grep "fail-count=" && complain 34 "Cluster resources' failures detected"
```

Comment:
@aspiers TBH I don't know how broad the error coverage of `crm_mon` is, I was expecting to catch all  cases. As Dirk said, it's the first step, we can tune it later.